validator/main.py

```diff
@@ -89,30 +89,17 @@ def chunking_function(self, chunk: str):
     def __init__(
         self,
         pii_entities: Union[str, List[str], None] = None,
+        use_local: bool = False,
```

Comment:
Move this below `on_fail` to preserve argument positions or else this could be breaking for existing users.

Also should `use_local` be optional?